x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -54,6 +58,10 @@ patterns. It also includes the authority to grant the privileges necessary to
 manage follower indices and auto-follow patterns. This privilege is necessary
 only on clusters that contain follower indices.
 
+`manage_enrollment`::
+The privilege to call the create enrollment token API and to call
+the Enroll Node and Enroll Token API. `enroll` is also included in `manage_enrollment`.
```

Comment:
We don;t have an `Enroll Token` API, right ? 

Also, maybe "is implicitly granted by" instead of "is also included in" ?